great_expectations/rule_based_profiler/parameter_builder/regex_pattern_string_parameter_builder.py

```diff
@@ -134,9 +135,21 @@ def _build_parameters(
             parameters=parameters,
         )
 
+        if not (
+            isinstance(metric_computation_result.metric_values, list)
+            and len(metric_computation_result.metric_values) == 1
+        ):
+            raise ge_exceptions.ProfilerExecutionError(
```

Comment:
When does this occur? Is there a way to make this code path unreachable with earlier checks?

Reply:
> When does this occur? Is there a way to make this code path unreachable with earlier checks?

@donaldheppner This never occurs (just did this check for defensive programming).